DSharpPlus/Entities/Guild/DiscordGuild.cs

```diff
@@ -924,6 +924,22 @@ public async Task<IReadOnlyList<DiscordRole>> ModifyRolePositionsAsync(IDictiona
         this._roles = new(returnedRoles.Select(x => new KeyValuePair<ulong, DiscordRole>(x.Id, x)));
         return returnedRoles;
     }
+    
+    /// <summary>
+    /// Removes a specified member from this guild.
+    /// </summary>
+    /// <param name="member">Member to remove.</param>
+    /// <param name="reason">Reason for audit logs.</param>
+    public async Task RemoveMemberAsync(DiscordMember member, string reason = null)
+        => await this.Discord.ApiClient.RemoveGuildMemberAsync(this.Id, member.Id, reason);
+
+    /// <summary>
+    /// Removes a specified member by ID.
+    /// </summary>
+    /// <param name="user_id">ID of the user to remove.</param>
+    /// <param name="reason">Reason for audit logs.</param>
+    public async Task RemoveMemberAsync(ulong user_id, string reason = null)
```

Comment:
Should be `userId`

Reply:
idk i followed how other methods are written

Reply:
it should be `userId`. the old code is wrong and going to be updated in time

Reply:
k